tests/test1.c

```diff
@@ -179,6 +179,9 @@ int main(int argc, char **argv)
 	printf("my_int=%d\n", json_object_get_int(my_int));
 	printf("my_int.to_string()=%s\n", json_object_to_json_string(my_int));
 
+	my_null = json_object_new_null();
```

Comment:
It seems a bit silly to write these tests pretending that we actually have a real object here, but as long as it works, ok.